### ISSUE TITLE
Crandall - ngsQC and assemblyQC for Pseudomonas aeruginosa - FDA ARGOS BioProject 

### ISSUE THREAD

**Issue by Jgergely11** [FDA-ARGOS/data.argosdb#294]
- [x] Please complete ngsQC for the following organisms and all SRRs associated with each Biosample using the HIVE ngsQC protocol.
- [x] Please complete assemblyQC for the following genome assembly IDs
- [x] Once data is ready to be added to datasets, Check with @Jgergely11 for ngsQC and @penningtonea for assemblyQC

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/jgerg/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/jgerg/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Organism | Bioproject | Biosample | Genome Assembly ID
-- | -- | -- | --
Pseudomonas aeruginosa | PRJNA231221 | SAMN16357210 | GCA_016126955.1
Pseudomonas   aeruginosa | PRJNA231221 | SAMN11056482 | GCA_006364735.1
Pseudomonas   aeruginosa | PRJNA231221 | SAMN10346774 | GCA_006364795.1
Pseudomonas   aeruginosa | PRJNA231221 | SAMN03996267 | GCA_001471435.2
Pseudomonas   aeruginosa | PRJNA231221 | SAMN06173353 | GCA_002206285.2
Pseudomonas   aeruginosa | PRJNA231221 | SAMN03996297 | GCA_001471355.2
Pseudomonas   aeruginosa | PRJNA231221 | SAMN10163178 | GCA_003938805.1
Pseudomonas   aeruginosa | PRJNA231221 | SAMN10163227 | GCA_003812165.1
Pseudomonas   aeruginosa | PRJNA231221 | SAMN10163177 | GCA_003812905.1
Pseudomonas   aeruginosa | PRJNA231221 | SAMN10163195 | GCA_003812885.1
Pseudomonas   aeruginosa | PRJNA231221 | SAMN10163194 | GCA_003813025.1
Pseudomonas   aeruginosa | PRJNA231221 | SAMN10163199 | GCA_003813005.1

</body>

</html>

**Comment by penningtonea** [comment]
Nucleotide Refseq: NC_002516.2
Assembly Refseq: GCF_000006775.1

**Comment by emdelattre** [comment]
@Jgergely11 the ngsQC for May is done and ready for review. Thanks!

**Comment by Jgergely11** [comment]
@emdelattre Thanks! Can you add the data to the V1.3 version of the google doc please?

**Comment by emdelattre** [comment]
@Jgergely11 I'm currently having issues locating the V1.3 version of the Google doc, would you be able to provide a link to the doc? So sorry about that.

**Comment by Jgergely11** [comment]
@emdelattre I didn't realize you didnt have access, just added you.

**Comment by emdelattre** [comment]
@Jgergely11 Thank you, I just got the email. I tried opening it and was denied access, so I just requested it.

**Comment by emdelattre** [comment]
@Jgergely11 I just checked and I still can't access the folder. I requested access again. Sorry about this.

**Comment by emdelattre** [comment]
@Jgergely11 I got access and put the data into the v1.3 document. Please let me know if there are any issues. Thanks again for your help!

**Comment by Jgergely11** [comment]
@emdelattre thanks for adding the data. Overall it looks pretty good, but there are a few formatting issues to take care of.

- Remove the empty rows between SRRs
- For these organisms there is no infraspecific name so those values can be deleted
- In the 'lineage' column replace all | with ;    -Just find and replace within that column because there are other columns where | is needed
- "schema_version" and "analysis_platform_object_id" it looks like the values were added sequentially instead of just copying down the column. The schema version should stay the same throughout the sheet (V1.3), and the object id should be consistent for each SRR
- "analysis_platform" should be HIVEAWS not HIVE AWS
- "sra_run_id" - remove the hyperlink
- There should not be any data in "num_reads_unique" - the output from HIVE is just the total number of reads, not unique reads. Just delete all those values
- There are some cells that have non-integer values from the HIVE output. Any time that happens it needs to be changed to a blank cell. I highlighted a few examples that I saw with SRR8179113.

Let me know if you have any questions
